src/app/pages/about/about.component.css

```
@@ -0,0 +1,12 @@
+#title {
+  text-align: center;
+  font-size: 5em;
+}
+
+#meet-the-team > div > div {
+  /* For all items within each peron's row -- copy of vertical-center class in main.css */
+  display: flex; 
+  flex-direction: column; 
+  justify-content: center; 
+  overflow: auto;
+}
```

No newline at end of file
```suggestion
}

```